src/Factory.php

```diff
@@ -89,7 +89,7 @@ protected function _mergeSeeds(...$seeds)
                 if ($obj !== null) {
                     continue; // legacy behaviour
 
-                    throw new \Exception('Two or more objects specified as seed.');
+                    //throw new \Exception('Two or more objects specified as seed.');
```

Comment:
no :)

Reply:
that exception will never throw.

Reply:
so we keep it until we remove `                    continue; // legacy behaviour`

keep to still support refactoring!